packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -68,6 +75,11 @@ export interface ModelsTreeProps extends BaseFilterableTreeProps {
    * Custom visibility handler.
    */
   modelsVisibilityHandler?: ModelsVisibilityHandler | ((props: ModelsVisibilityHandlerProps) => ModelsVisibilityHandler);
+  /**
+   * Flag that determines if hierarchy level filtering will be enabled for this tree.
+   * @beta
+   */
+  isHierarchyFilteringEnabled?: boolean;
```

Comment:
Just noticed... this should be named `isHierarchyLevelFilteringEnabled` (to avoid confusion with overall hierarchy filtering)